app/forms/upload_labware_form.rb

```diff
@@ -6,9 +6,9 @@
 class UploadLabwareForm
```

Comment:
Maybe we could set and check the user in the manifest uploader form. That way it is all in one place.